public/home.html

```diff
@@ -64,18 +65,29 @@
     <h4 id="title" onclick="openTab(event, 'grades')">Aspine</h4>
   </div>
 
+  <!-- SIDENAV STUFF -->
+  <div id="sidenav" class="sidenav">
+    <!-- switch-exempt makes the switch out of navbar pass over this -->
+    <button onclick="closeSideNav()" class="tablinks switch-exempt" style="margin: 0px;width: 270px; height: 44px; background-color: var(--lightgray1)">
+      <i class="fa fa-times fa-2x" style="position: absolute; top: 18px; left: 18px; font-size: 1.3em; color: #000""></i>
```

Comment:
Remove the double closing quote - while it may still work in some browsers, I think it's still a syntax error and it confuses some syntax highlighters
```suggestion
      <i class="fa fa-times fa-2x" style="position: absolute; top: 18px; left: 18px; font-size: 1.3em; color: #000"></i>
```